surveys/typeform.py

```diff
@@ -104,6 +106,8 @@ def sync_learner_responses():
             study_group = StudyGroup.objects.get(uuid=study_group_id)
         except ObjectDoesNotExist as e:
             logger.debug('Study group with ID does not exist', e)
+        except ValidationError as e:
+            logger.debug('UUID is not valid', e)
```

Comment:
On typeform we have a few empty responses where the UUID is saved as `xxxxx`. I think it happens when someone fills out the form from the editing page on Typeform? Anyway it was throwing this error:
```
django.core.exceptions.ValidationError: ["'xxxxx' is not a valid UUID."]
```
So this catches the error, I ran the task locally and it got all the responses.